poll/poll.py

```diff
@@ -452,26 +472,26 @@ def vote(self, data, suffix=''):
         result = {'success': False, 'errors': []}
         old_choice = self.get_choice()
         if (old_choice is not None) and not self.private_results:
-            result['errors'].append('You have already voted in this poll.')
+            result['errors'].append(self.ugettext('You have already voted in this poll.'))
             return result
         try:
             choice = data['choice']
         except KeyError:
-            result['errors'].append('Answer not included with request.')
+            result['errors'].append(self.ugettext('Answer not included with request.'))
             return result
         # Just to show data coming in...
         try:
             OrderedDict(self.answers)[choice]
         except KeyError:
-            result['errors'].append('No key "{choice}" in answers table.'.format(choice=choice))
+            result['errors'].append(self.ugettext('No key "{choice}" in answers table.').format(choice=choice))
```

Comment:
@asadiqbal08 This is missing a translator comment, something like `# Translators: {choice} uniquely identifies a specific answer belonging to a poll or survey`.